packages/env/lib/wordpress.js

```diff
@@ -47,6 +47,27 @@ async function makeContentDirectoriesWritable(
 	);
 }
 
+/**
+ * Makes wp-config.php owned by www-data so that WordPress can work with the
+ * file.
+ *
+ * @param {WPEnvironment} environment The environment to check. Either 'development' or 'tests'.
+ * @param {WPConfig}      config      The wp-env config object.
+ */
+async function makeConfigWritable(
```

Comment:
I wish we could combine this with `makeContentDirectoriesWritable`, but I see why they have to be separate.

Reply:
The only way would be to call `makeContentDirectoriesWritable` always (where I called `makeConfigWritable`), instead of calling it conditionally as it is now. But apparently that would not be good for wordpress installations that are not in docker volumes (according to a comment where `makeContentDirectoriesWritable` is used).

Reply:
I forget the details, but there was definitely a bug that conditional resolves